CODEOWNERS

```diff
@@ -5,14 +5,5 @@
 
 # The '*' pattern is global owners.
 
-* @djack @kroenlein @maxhutch
-
-
-
-
-
-
-
-
-
+*  @CitrineInformatics/gemd-pr-reviewers
```

Comment:
Should there be two groups (one for docs, one for python) or are they the same?

Reply:
for now they are the same, and I mined the contributors to the repos to populate them.